bin/up

```diff
@@ -24,8 +33,12 @@ function check_config() {
 }
 
 function __main__() {
+  if [[ "${1:-null}" == "help" ]] || [[ "${1:-null}" == "--help" ]]; then
+    usage
+    exit
+  fi
   check_config
-  exec "$TOOLKIT_ROOT/bin/docker-compose" up
+  exec "$TOOLKIT_ROOT/bin/docker-compose" up "$@"
```

Comment:
I've always been bothered by `"$@"` because of how the quotes work, but that's not a problem here. :-)

Reply:
I agree that it's deeply counter-intuitive. It's worth checking out `shellcheck`, which can lint these kinds of scripts. It recommends using the quotes, and I'm inclined to obey the machine ;)

Reply:
In fact, we have a helper in `bin/dev/lint`, which runs shellcheck on the `bin/` directory.

EDIT: actually, I now find that the `lint` script is showing some problems in the code base, will fix :sweat_smile: